package.json

```diff
@@ -23,6 +23,7 @@
   "devDependencies": {
     "chai": "^3.2.0",
     "coveralls": "^2.11.2",
+    "eslint": "^1.3.1",
```

Comment:
Unfortunately no. At first, Peasant was handling this without adding the dependency, but then we run into an [issue](https://github.com/danielgtaylor/peasant/commit/6f7d7fbb6cb8878a7373777d93274cdd4a1f8253#diff-04c6e90faac2675aa89e2176d2eec7d8R61) with text editors that lint as you type or when saving files. The text editor plugins will look for a global eslint or a locally installed one as a **direct dependency** of your package. They had no way of finding the ESLint installed inside of Peasant, would switch to the globally installed one and then be unable to find the `eslint-config-airbnb` package that was installed locally.

You *could* expect a user to install both ESLint and `eslint-config-airbnb` globally, but then you lose control over the ESLint and config versions used on a per-project basis (and maybe some users can't install globally). Ultimately I went with what you see here today - your package depends on `peasant`, `eslint`, and `eslint-config-airbnb`. Everything else stays inside Peasant, so it still saves you quite a few direct dependencies and configuration.

Part of the goal of Peasant is to simplify and speed up development of ES6 modules. That means making the `package.json` simpler, but it also means trying to make running tests and such fast. Part of the recommended flow with Peasant now includes linting as you work on files rather than during `npm test`, so that testing can focus purely on running the tests. Let the CI system and prepublish command handle linting while your text editor does it for you locally, letting you run tests faster (which was a major pain point when initially exploring ES6 support).